yt_dlp/extractor/weyyak.py

```diff
@@ -0,0 +1,55 @@
+import re
+
+from .common import InfoExtractor
+from ..utils import traverse_obj
+
+
+class WeyyakIE(InfoExtractor):
+    _VALID_URL = r'https?://weyyak\.com/(?P<lang>\w+)/(?:player/)?(?P<type>episode|movie)/(?P<id>\d+)'
+    _TESTS = [
+        {
+            'url': 'https://weyyak.com/en/player/episode/1341952/Ribat-Al-Hob-Episode49',
+            'md5': '0caf55c1a615531c8fe60f146ae46849',
+            'info_dict': {
+                'id': '1341952',
+                'ext': 'm3u8',
+                'title': 'Ribat Al Hob',
+                'duration': 2771,
+            },
+        },
+        {
+            'url': 'https://weyyak.com/en/movie/233255/8-Seconds',
+            'md5': 'fe740ae0f63e4d1c8a7fc147a410c564',
+            'info_dict': {
+                'id': '233255',
+                'ext': 'm3u8',
+                'title': '8 Seconds',
+                'duration': 6490,
+            },
+        },
+    ]
+
+    def _real_extract(self, url):
+        _id, _lang, _type = self._match_valid_url(url).group('id', 'lang', 'type')
+
+        path = 'episode/' if _type == 'episode' else 'contents/moviedetails?contentkey='
+        video_info = self._download_json(
+            f'https://msapifo-prod-me.weyyak.z5.com/v1/{_lang}/{path}{_id}', _id,
+            headers={'Content-Type': 'application/json'})
```

Comment:
since this is not a POST request, the `content-type` header is not needed
```suggestion
        video_info = self._download_json(
            f'https://msapifo-prod-me.weyyak.z5.com/v1/{_lang}/{path}{_id}', _id)
```